client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Submission Draft page
![Screenshot of Submission Draft page showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/eef495d5-7e7a-4ef8-b7f7-eebc547f1afa)

Reply:
These now appear as 16px.